pantos/client/library/entitites.py

```diff
@@ -0,0 +1,101 @@
+"""Module that defines the Pantos client library entities.
+
+"""
+import dataclasses
+import enum
+import uuid
+
+from pantos.common.blockchains.enums import Blockchain
+from pantos.common.servicenodes import ServiceNodeTransferStatus
+from pantos.common.types import BlockchainAddress
+
+
+class DestinationTransferStatus(enum.Enum):
+    """Enumeration of the possible destination transfer statuses.
+
+    """
+    UNKNOWN = 0
+    SUBMITTED = 1
+    CONFIRMED = 2
+
+
+@dataclasses.dataclass
+class TokenTransferInfo:
```

Comment:
All the information here is purely service-node-related. Maybe we should make that clear by actually calling the dataclass something like `ServiceNodeTaskInfo`.